spec/services/activitypub/fetch_remote_key_service_spec.rb

```diff
@@ -7,9 +7,11 @@
 
   let(:webfinger) { { subject: 'acct:alice@example.com', links: [{ rel: 'self', href: 'https://example.com/alice' }] } }
 
+  # rubocop:disable Layout/LineLength
```

Comment:
We could probably chop this one up on the newline chars and heredoc the whole thing in the let assignment.